CHANGELOG.md

```diff
@@ -56,6 +56,7 @@
 - The <i class="fa fa-picture-o"/> (add image) and <i class="fa fa-link"/> (add link) toolbar buttons, put selected links directly in the `()` instead of the `[]` part of the generated markdown
 - The help dialog has multiple tabs, and is a bit more organized.
 - Use KaTeX instead of MathJax. ([Why?](https://community.codimd.org/t/frequently-asked-questions/190))
+- The CLI and 3rd-party-clients now rely on access tokens, they can be managed in the user profile.
```

Comment:
The changelog is meant to be read by humans. Therefore, I guess it's useful to include a small piece of context.

Reply:
Yes, but because of the `now` (please try to avoid this word in change logs anyway) it sounds like `This release of the frontend makes the cli rely on access tokens`